docker-compose.yml

```diff
@@ -7,5 +7,6 @@ services:
       dockerfile: Dockerfile.dev
     volumes:
       - .:/iris-messenger
+      - /iris-messenger/node_modules
```

Comment:
We have to mount the node_modules in the container because it will be replaced by the first volume (which defeats the purpose of the container).